lib/BackupPC/CGI/Metrics.pm

```diff
@@ -113,10 +113,12 @@ sub action
     # Host metrics
     #
     foreach my $host ( GetUserHosts(1) ) {
-        my($fullAge, $fullCount, $fullDuration, $fullRate, $fullSize, $incrAge, $incrCount, $incrDuration);
+        my($fullCount, $fullDuration, $fullRate, $fullSize);
+        my($incrCount, $incrDuration, $incrRate, $incrSize);
+        my($lastFullBackup, $lastIncrBackup);
 
         $fullCount = $incrCount = 0;
-        $fullAge   = $incrAge   = $fullDuration = $incrDuration = -1;
+        $$lastFullBackup->{StartTime} = $$lastIncrBackup->{StartTime} = -1;
```

Comment:
As the following references of startTime have a lower 's', these should also start with a lower 's' and not with a capital 'S'. Because of this, the following warnings are caused multiple times, depending on the number of hosts and backups:
```
backuppc  | [Tue Aug  6 07:26:59 2024] BackupPC_Admin: Use of uninitialized value in numeric gt (>) at /usr/local/BackupPC/lib/BackupPC/CGI/Metrics.pm line 133.
backuppc  | [Tue Aug  6 07:26:59 2024] BackupPC_Admin: Use of uninitialized value in numeric lt (<) at /usr/local/BackupPC/lib/BackupPC/CGI/Metrics.pm line 138.
```